src/apps/arth/arth.definition.ts

```diff
@@ -0,0 +1,45 @@
+import { Register } from '~app-toolkit/decorators';
```

Comment:
We recently moved the definition to db and removed definition files, meaning this file won't be necessary and you will be able to remove it